### PR TITLE
docs(astro-uxds): updated implementation text on theme page

### PR DESCRIPTION
**Pull request by Paterma**
## Brief Description

Updated theme page text under "Implementations"

## JIRA Link

(https://rocketcom.atlassian.net/browse/ASTRO-3924)

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [ ] Regressions are passing and/or failures are documented
- [ ] Changes have been checked in evergreen browsers

### CODE REVIEW

File: packages/astro-uxds/_content/design-guidelines/theme.md [hunk 1]
```diff
@@ -49,4 +49,5 @@ While not enabled by default at this time, light and dark themes can be swapped
 
 ## Implementation
 
-Astro applications using the Astro CSS Library will default to the dark theme with no additional involvement by the developer. To enable the light theme with the Astro CSS Library, the `<body>` element of the application should be given a light-theme class (e.g., `<body class=“light-theme”>`. For all other applications, refer to the [Color Palettes](/design-guidelines/color) for specific hex/rgb values in light and dark UIs.
+When using the Astro Web Components, Astro applications will default to the dark theme with no additional involvement by the developer. To enable the light theme, the <body> element of the application should be given a light-theme class (e.g., <body class=“light-theme”>.
+For all other applications, refer to our [Design Tokens](/design-tokens/getting-started).
```
Comment (micahjones13) [inline packages/astro-uxds/_content/design-guidelines/theme.md:53]: This should probably not be on it's own line - that's my b. Could you move it to be in line with 52?

**Review by micahjones13 — CHANGES_REQUESTED** [review]

File: packages/astro-uxds/_content/components/clock.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ theme: true
 
 # Clock
 
-Clock shows the current time and, optionally, Julian date, AOS, and LOS timers. It will typically be positioned on the Global Status Bar.
+Clock shows the current time and optional date, AOS, and LOS timers. It will typically be positioned on the Global Status Bar.
```
Comment (micahjones13) [inline packages/astro-uxds/_content/components/clock.md:16]: May be some branch leak here from the clock change - that should be fine in this case though.

Comment (Paterma) [reply packages/astro-uxds/_content/components/clock.md:16]: I un-did this change? Sorry about that- not sure how it was back in there.

Comment (micahjones13) [reply packages/astro-uxds/_content/components/clock.md:16]: magic 🎉

**Review by micahjones13 — APPROVED** [review]

**Pull request merged**